librocksdb_sys/build.rs

```diff
@@ -24,6 +25,49 @@ use std::{env, str};
 // See https://github.com/gnzlbg/jemallocator/blob/bfc89192971e026e6423d9ee5aaa02bc56585c58/jemalloc-sys/build.rs#L45
 const NO_JEMALLOC_TARGETS: &[&str] = &["android", "dragonfly", "musl", "darwin"];
 
+// Generate the bindings to rocksdb C-API.
+// Try to disable the generation of platform-related bindings.
+fn bindgen_rocksdb(file_path: &PathBuf) {
+    let bindings = bindgen::Builder::default()
+        .header("crocksdb/crocksdb/c.h")
+        .ctypes_prefix("libc")
+        .generate()
+        .expect("unable to generate rocksdb bindings");
+
+    bindings
+        .write_to_file(file_path)
+        .expect("unable to write rocksdb bindings");
+}
+
+// Determine if need to update bindings. Supported platforms do not
+// need to be updated by default unless the UPDATE_BIND is specified.
+// Other platforms use bindgen to generate the bindings every time.
+fn config_binding_path() {
+    let file_path: PathBuf;
+    println!("cargo:rerun-if-changed=bindings/x86_64-unknown-linux-gnu-bindings.rs");
```

Comment:
Please add `cargo:rerun-if-env-changed=UPDATE_BIND`, and remove this cause it will forbid rerun when other files changed in ` librocksdb_sys/*`